.buildconfig/windows-latest.yml

```diff
@@ -0,0 +1,25 @@
+name: windows-latest
```

Comment:
Is this file identical to the OSX one? Could we have a file with doc requirements and one without?
Maybe having two or three files doesn't change much, and in the future we may want different files of windows and osx?
So it's probably ok to keep the 3 files like it is.
(sorry for the noise, just thinking aloud here)

Reply:
We had only 2 files in the past and it always was a mess.